contribute-docs/style-guide/voice-tone.md

```diff
@@ -0,0 +1,365 @@
+| Human, plainspoken | Academic or too casual | Be conversational, yet professional. Write close to how you'd talk.                        |
+| Empathetic, caring | Cutesy                 | Write what matters to users. Consider the impact of your content on user experience.                              |
+| Inclusive, open    | Exclusive              | Write for all audiences. Don't use terms that might discriminate or offend.            |
+| User-focused       | User-obsessed          | Use `you` and `yours` by default, but not when the user doesn't directly own what's described. |
+
+### We help users get their job done
+
+With timely, succinct, and impactful content, we can make our users' jobs easier and help them be more efficient. 
+
+| ✔️ We are... | ❌ But not... | How?                                                                                    |
+|-------------|--------------|-----------------------------------------------------------------------------------------|
+| Informative | Overly wordy | Ensure all content is relevant, serves a purpose, and is not just decoration.                         |
+| Transparent | Disorganized | Provide the right details at the right time—not too soon, not too late. Be accurate.                |
+| Succinct    | Vague        | Shorten the content to something meaningful and scannable, without sacrificing clarity. |
+
+
+### We inspire users to think big
+
+Through our content, we empower users to do more and to dive into their next actions. We help them achieve their potential and trust them to forge their best path forward.
+
+| ✔️ We are...     | ❌ But not...               | How?                                                                               |
+|-----------------|----------------------------|------------------------------------------------------------------------------------|
+| Inspiring       | Arrogant                   | Use words that motivate users and help them to exceed their limits.                |
+| Confident       | Pretentious, controlling   | Be definite and trustworthy. Delete unnecessary wording. Don't oversell.           |
+| Forward-looking | Hand-holding, prescriptive | Help users connect the dots and see the big picture, while giving them the choice. |
+
+
+## Tone
+
+Our products and solutions are very technical, data-oriented, and they're used globally by people from different cultures and personalities. 
+These considerations, amongst others, limit the spectrum of tones we use to some extent, but not entirely. 
+ 
+During the users' journey with Elastic, there are times for excitement and discovery, but also times for help, support, and empathy.
+
+**Our tone scale**
+![Tone scale](./images/tones.png)
+
+### Stimulating
+
+:::::{tab-set}
+::::{tab-item} Attributes
+    We're proud to show new and cool stuff. Be:
+
+    - Motivational
```

Comment:
These bullets aren't working as-is. Maybe due to spacing? There are lots of similar breaking bullets on this page,  but I won't submit suggestions for all of them
```suggestion
 - Motivational
```